docs/messaging/messaging-spans.md

```diff
@@ -122,12 +120,6 @@ First, the lower-level receiving of a message at a consumer, and then the logica
 Often, the waiting for a message is not particularly interesting and hidden away in a framework that only invokes some handler function to process a message once one is received
 (in the same way that the listening on a TCP port for an incoming HTTP message is not particularly interesting).
 
-### Conversations
-
-In some messaging systems, a message can receive one or more reply messages that answers a particular other message that was sent earlier. All messages that are grouped together by such a reply-relationship are called a *conversation*.
-The grouping usually happens through some sort of "In-Reply-To:" meta information or an explicit *conversation ID* (sometimes called *correlation ID*).
-Sometimes a conversation can span multiple message destinations (e.g. initiated via a topic, continued on a temporary one-to-one queue).
-
```

Comment:
I have a slight preference of keeping this as a definition here instead of moving it into a footnote. We're using the term conversation e. g. in the following section about "Temporary and anonymous destinations", and it would be counter-intuitive to have it defined in a footnote further below.

Reply:
@pyohannes Thanks for that feedback, I was not sure with that as well.

The problem we have is that two attributes had links to these sections in their definition. Now, that we are moving the definitions to the registry it feels wrong to have links to markdown files outside of the registry. Any ideas how we could solve that? Just removing the links?

Reply:
Good question, I don't have an ideal solution.

I'd have a preference for leaving the "Conversation" definition as it is, and removing the link from the attribute `brief`.

Reply:
@pyohannes I reverted the change regarding the sections, PTAL